server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/QProfileWsSupport.java

```diff
@@ -63,12 +63,22 @@ public OrganizationDto getOrganizationByKey(DbSession dbSession, @Nullable Strin
       "No organizationDto with key '%s'", organizationOrDefaultKey);
   }
 
+  /**
+   * @deprecated provide orgnization
+   */
+  @Deprecated
   public void checkQProfileAdminPermission() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)